src/components/Download.tsx

```diff
@@ -38,54 +40,96 @@ function Download() {
             Description: column.description || '',
           };
 
-          if (column.standardizedVariable) {
-            dictionaryEntry.Annotations = {};
+          // Description of levels always included for the BIDS section
+          if (column.dataType === 'Categorical' && column.levels) {
+            dictionaryEntry.Levels = Object.entries(column.levels).reduce(
+              (levelsObj, [levelKey, levelValue]) => ({
+                ...levelsObj,
+                [levelKey]: {
+                  Description: levelValue.description || '',
+                },
+              }),
+              {} as { [key: string]: { Description: string } }
+            );
+          }
 
-            dictionaryEntry.Annotations.IsAbout = {
-              Label: column.standardizedVariable.label,
-              TermURL: column.standardizedVariable.identifier,
+          if (column.dataType === 'Continuous' && column.units !== undefined) {
+            dictionaryEntry.Units = column.units;
+          }
+
+          if (column.standardizedVariable) {
+            dictionaryEntry.Annotations = {
+              IsAbout: {
+                TermURL: column.standardizedVariable.identifier,
+                Label: column.standardizedVariable.label,
+              },
             };
 
+            // Handle level terms only for standardized columns
+            if (column.dataType === 'Categorical' && column.levels) {
+              // Add TermURL to root Levels if present
+              dictionaryEntry.Levels = Object.entries(column.levels).reduce(
+                (updatedLevels, [levelKey, levelValue]) => ({
+                  ...updatedLevels,
+                  [levelKey]: {
+                    ...updatedLevels[levelKey],
+                    ...(levelValue.termURL ? { TermURL: levelValue.termURL } : {}),
+                  },
+                }),
+                dictionaryEntry.Levels || {}
+              );
+
+              // Create Annotations.Levels for terms
```

Comment:
Similar here, needs to be clearer. I think you mean that **Annotations** Levels are only created for columns mapped to standardized variables, and where the unique values were mapped to standardized terms.

Reply:
i.e. the point of the comment should be to explain *why* you are doing this and *why* it is different for the different scenarios - not what you are doing.